src/pages/AdminDashboard.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState, useEffect} from 'react';
+import { connect } from "react-redux";
+import { fetchAllApps, allApps, isLoading } from "../redux/actions/fetchApps.actions.js";
+import AppsList from "../components/AppsList.js"
+import { request } from "../backend-request";
+
+// export const AdminDashboard = ({isLoading, allApps, fetchAllApps}) => {
+export const AdminDashboard = () => {
+  const [allApps, setAllApps] = useState([])
+  // useEffect(() => {
```

Comment:
As I mentioned in some other places as well.. please remove commented out codes as we dont use those